internal/benchmark/bench.cu

```diff
@@ -4,9 +4,11 @@
 #include <thrust/sort.h>
 #include <thrust/reduce.h>
 #include <thrust/scan.h>
+#include <thrust/shuffle.h>
 
 #include <algorithm>
 #include <numeric>
+#include <random>
```

Comment:
Should we guard this against pre-C++11? (Not sure if that makes sense given that we're deprecating that.)